static/js/revision_diff.js

```diff
@@ -0,0 +1,71 @@
+let buttons = document.querySelectorAll("td input");
```

Comment:
Missing `"use strict";`
Missing comments

Also, should probably have this entire thing inside a closure

Reply:
Fixed in latest commit.